src/main/target/LUX_RACE/target.h

```diff
@@ -27,7 +27,7 @@
 #endif
 
 // Removed to make the firmware fit into flash (in descending order of priority):
-//#undef USE_RTC_TIME
+#undef USE_RTC_TIME
```

Comment:
Please keep pull requests focused on one change per pull request, and do not use them to change unrelated features.
Removing existing features from targets will be reported as 'bug' by users using these features, and having to dig through pull requests that don't clearly specify that they are doing this will just make this harder to support.

The easiest way to address this is to add an `#undef USE_D_CUT` to this target. Alternatively, features will have to be pruned from this target in a separate pull request.

Reply:
done